packages/compiler/test/checker/relation.test.ts

```diff
@@ -2,9 +2,9 @@ import { deepStrictEqual, ok, strictEqual } from "assert";
 import { Model } from "../../core/index.js";
 import {
   BasicTestRunner,
+  DiagnosticMatch,
```

Comment:
All those import change have made capitalize case ahead of  lower when sorting.
I think this is maybe better this way but does add a decent amount of changes. We unfortunatelly can't upgrade if we don't take this change right now as there is no option exposing this configuration. Linked an issue in description for the prettier organize import plugin

Reply:
It's unfortunate to have all the churn, but it's prettier in my subjective opinion 😆